### PR TITLE
Feature/OMEGA-117 poweroutage pin component

### PR DESCRIPTION
**Pull request by erlbe**
OMEGA-117

## What is the reason for this change?

Use pins with scaling size to display power outages in map.

### What is the expected result?

A pin which scales according to number of affected customers.

## How is it solved?

Created a Pin component with accompanied style.

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |       |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?
- [ ] Verify that pin is present above map in /strombruddskartv2 page.

### CODE REVIEW

File: packages/poweroutage/src/components/Pin/Pin.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  return (
+    <div
+      className={classnames(css.pin, {
+        [css.pinPlanned]: type === Type.Planned,
+      })}
+      onClick={onClick}
+      style={{
+        height: `${outerDiameter}px`,
+        width: `${outerDiameter}px`,
```
Comment (corneliusgdahling) [inline packages/poweroutage/src/components/Pin/Pin.tsx:50]: use spaceCalc here?

File: packages/poweroutage/src/components/Pin/Pin.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      }}
+    >
+      <div
+        className={classnames(css.innerCircle, className)}
+        style={{
+          height: `${innerDiameter}px`,
+          width: `${innerDiameter}px`,
```
Comment (corneliusgdahling) [inline packages/poweroutage/src/components/Pin/Pin.tsx:57]: spaceCalc

**Review by corneliusgdahling — CHANGES_REQUESTED** [review]
Just some small comments, then we can :shipit: !

File: packages/poweroutage/src/components/Pin/Pin.module.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    background-color: map-get($colors, 'orangeSignal');
+  }
+
+  .innerCircle {
+    background-color: map-get($colors, 'black');
+    border-radius: 50%;
+
+    .innerText {
+      color: map-get($colors, 'white');
+      font-size: '20px';
```
[the remark this answers is not in the document]
Comment (erlbe) [reply packages/poweroutage/src/components/Pin/Pin.module.scss]: Fixed 👍

File: packages/poweroutage/src/components/Pin/Pin.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        }}
+      >
+        <div
+          className={classnames(css.innerText, className)}
+          style={{
+            lineHeight: `${innerDiameter}px`,
+          }}
+        >
+          {affectedCustomers}
+        </div>
```
[the remark this answers is not in the document]
Comment (erlbe) [reply packages/poweroutage/src/components/Pin/Pin.tsx]: Fixed 👍

File: packages/poweroutage/src/components/Pin/Pin.tsx (hunk 1 above)
Comment (erlbe) [reply packages/poweroutage/src/components/Pin/Pin.tsx]: Fixed 👍

File: packages/poweroutage/src/components/Pin/Pin.tsx (hunk 2 above)
Comment (erlbe) [reply packages/poweroutage/src/components/Pin/Pin.tsx]: Fixed 👍

**Review by corneliusgdahling — APPROVED** [review]

**Pull request merged**